components/internal/PermitHolderInfoCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { GridItem, Flex, Text, Link, Spacer, GridItemProps } from '@chakra-ui/react'; // Chakra UI
+import { MouseEventHandler, ReactNode } from 'react'; // React
+
+type PermitHolderInfoCardProps = GridItemProps & {
+  children: ReactNode;
+  header: ReactNode;
+  updated?: boolean;
+  handleEdit: MouseEventHandler;
+};
+
+/**
+ * Custom Card component with styling.
+ * @param props - Props
+ * @returns custom Card.
+ */
+export default function PermitHolderInfoCard(props: PermitHolderInfoCardProps) {
+  const { children, header, updated, handleEdit } = props;
+  return (
+    <GridItem
+      display="flex"
+      flexDirection="column"
+      alignItems="flex-start"
+      padding="20px 24px 24px"
+      background="white"
+      border="1px solid"
+      borderColor="border.secondary"
+      boxSizing="border-box"
+      borderRadius="8px"
+      {...props}
+    >
+      <Flex w="100%" justifyContent="flex-start" alignItems="center">
+        {typeof header == 'string' ? (
+          <Text textStyle="display-small-semibold">{header}</Text>
```

Comment:
All of the `Text` elements are missing the `as` prop, please add it to each of them. You can see documentation about it here: https://chakra-ui.com/guides/as-prop. It basically helps browsers generate semantic html which helps screen readers work better. Checkout [verify-identity.tsx](https://github.com/uwblueprint/richmond-centre-for-disability/blob/6189557c1facd578ec1e9278c2fd5017ce8b80cb/pages/verify-identity.tsx) for examples.